backend/dataall/modules/metadata_forms/db/metadata_form_models.py

```diff
@@ -0,0 +1,149 @@
+from sqlalchemy import (
+    Column,
+    String,
+    Integer,
+    ForeignKey,
+    Boolean,
+    ForeignKeyConstraint,
+    PrimaryKeyConstraint,
+)
+from sqlalchemy.dialects.postgresql import ARRAY
+from sqlalchemy.orm import relationship, validates
+
+from dataall.base.db import Base, utils
+
+from dataall.modules.metadata_forms.db.enums import MetadataFormFieldType
+
+
+class MetadataForm(Base):
+    __tablename__ = 'metadata_form'
+    uri = Column(String, primary_key=True, default=utils.uuid('metadata_form'))
+    name = Column(String, nullable=False)
+    description = Column(String, nullable=False)
+    SamlGroupName = Column(String, nullable=False)
+    visibility = Column(String, nullable=False)  # enum MetadataFormVisibility
+    homeEntity = Column(String, nullable=True)
+
+
+class MetadataFormEnforcementRule(Base):
+    __tablename__ = 'metadata_form_enforcement_rule'
+    uri = Column(String, primary_key=True, default=utils.uuid('rule'))
+    metadataFormUri = Column(String, ForeignKey('metadata_form.uri'))
+    level = Column(String, nullable=False)  # enum MetadataFormEnforcementScope
+    entityTypes = Column(ARRAY(String), nullable=False)  # enum MetadataFormEntityTypes
+    severity = Column(String, nullable=False)  # enum MetadataFormEnforcementSeverity
+
+
+class MetadataFormField(Base):
+    __tablename__ = 'metadata_form_field'
+    metadataFormUri = Column(String, ForeignKey('metadata_form.uri'))
+    uri = Column(String, primary_key=True, default=utils.uuid('field'))
+    name = Column(String, nullable=False)
+    type = Column(String, nullable=False)  # enum MetadataFormFieldType
+    required = Column(Boolean, nullable=False)
+    glossaryTerm = Column(String, ForeignKey('term_link.linkUri'), nullable=True)
+    possibleValues = Column(ARRAY(String), nullable=True)
```

Comment:
how do we `possibleValues`?

Reply:
It's something that user enters upon creating the MF, when the value is restricted, but not the Glossary
E.G. "Is it used for customers issues?" , possible values: "yes, no, only with explicit approval from CEO"